error: to_string is not a member of boost

**Describe the bug**

Using Boost 1.89.  Current HEAD (6407d9c).  Error during build.
```
/build/vcmi-git/src/vcmi/scripting/erm/ERMInterpreter.cpp: In member function ‘std::string ERMConverter::Variable::str() const’:
/build/vcmi-git/src/vcmi/scripting/erm/ERMInterpreter.cpp:80:47: error: ‘to_string’ is not a member of ‘boost’
   80 |                                 return boost::to_string(boost::format("M['%s']") % macro);
      |                                               ^~~~~~~~~
```

Builds successfully with following change:
```
sed -E -e 's&(return) boost::to_string(.*);$&\1 \2.str();&' -i scripting/erm/ERMInterpreter.cpp
```

**Game logs**
N/A.  Problem is during build, not runtime.

**To Reproduce**
Steps to reproduce the behavior:

1. Clone repo, update submodules, install deps, etc.
2. Configure with: `ENABLE_ERM=ON`, `ENABLE_LUA=ON`
3. Build... see aforementioned error...

**Actual behavior**
Aforementioned error during build.

**Did it work earlier?**
Previously built 1.6.8 with Boost 1.88.

**Screenshots**
N/A.

**Version**
- OS: Arch Linux
- Version: HEAD 6407d9c.

**Additional context**
Building HEAD because of Boost 1.89.

--- a/scripting/erm/ERMInterpreter.cpp
+++ b/scripting/erm/ERMInterpreter.cpp
@@ -801,14 +801,14 @@ namespace ERMConverter
 						fmt % target % v.str() % opt;
 						put(fmt.str());
 					}
-					
+
 					for(int i = paramIndex; i < trig.params->size(); i++)
 					{
 						opt = std::visit(VR_X(), (*trig.params)[i]);
 						if(i > paramIndex) put(",");
 						put(opt);
 					}
-					
+
 					putLine(")");
 				}
 				break;
@@ -1183,7 +1183,7 @@ namespace ERMConverter
 		}
 		void operator()(const boost::recursive_wrapper<VNode> & opt) const;
 
-		void operator()(const VSymbol & opt) const 
+		void operator()(const VSymbol & opt) const
 		{
 			(*out) << "\"" << opt.text << "\"";
 		}
@@ -1589,7 +1589,7 @@ namespace VERMInterpreter
 		struct OptionConverterVisitor
 		{
 			VOption operator()(const boost::recursive_wrapper<ERM::TVExp>& cmd) const
-			{ 
+			{
 				return boost::recursive_wrapper<VNode>(VNode(cmd.get()));
 			}
 			VOption operator()(const ERM::TSymbol & cmd) const
@@ -1599,7 +1599,7 @@ namespace VERMInterpreter
 				else
 					return boost::recursive_wrapper<VNode>(VNode(cmd));
 			}
-			VOption operator()(const char & cmd) const 
+			VOption operator()(const char & cmd) const
 			{
 				return TLiteral(cmd);
 			}

